README.md

```diff
@@ -2,13 +2,56 @@
 
 A NestJS module for managing environment variables easily and securely.
 
+<hr/>
+
+### What Difference From [official library](https://docs.nestjs.com/techniques/configuration)
```

Comment:
Could you move this section below line 54?

It would be better for us to highlight the difference in `feature` section.

Reply:
ok got it

Reply:
reflected it